[DOCS] ServiceNow ITSM & SecOps Connector and Application Docs (#1188)
elastic/security-docs: docs/cases/cases-ui-integrations.asciidoc

. Go to *Cases* -> *Edit external connection*.
+
[role="screenshot"]
image::images/cases-ui-connector.png[Shows the page for creating connectors]
. From the *Incident management system* list, select *Add new connector*.
. Select one of these:
* *{sn}*: To send cases to {sn}

+

> IMPORTANT: If you've upgraded from {stack} version \<=7.15 to version >=7.16, you must install https://store.servicenow.com/sn_appstore_store.do#!/store/application/7148dbc91bf1f450ced060a7234bcb88[Elastic for ITSM] or https://store.servicenow.com/sn_appstore_store.do#!/store/application/2f0746801baeb01019ae54e4604bcb0f[Elastic for Security Operations (SecOps)] on your {sn} instance and complete additional configuration steps before creating a new {sn} ITSM or SecOps connector. If you don't, an error message displays and you cannot create the new connector. For more information, refer to {kibana-ref}/servicenow-sir-action-type.html[{sn} SecOps] and {kibana-ref}/servicenow-action-type.html[{sn} ITSM].

Reviewer: ```suggestion
IMPORTANT: If you've upgraded from {stack} version \<=7.15.0 to version >=7.16.0, you must install https://store.servicenow.com/sn_appstore_store.do#!/store/application/7148dbc91bf1f450ced060a7234bcb88[Elastic for ITSM] or https://store.servicenow.com/sn_appstore_store.do#!/store/application/2f0746801baeb01019ae54e4604bcb0f[Elastic for Security Operations (SecOps)] on your {sn} instance and complete additional configuration steps before creating a new {sn} ITSM or SecOps connector. If you don't, an error message displays and you cannot create the new connector. For more information, refer to {kibana-ref}/servicenow-sir-action-type.html[{sn} SecOps] and {kibana-ref}/servicenow-action-type.html[{sn} ITSM].
```

. From the *Incident management system* list, select *Add new connector*.
. Select one of these:
* *{sn}*: To send cases to {sn}

+

> IMPORTANT: Before creating a new {sn} ITSM or SecOps connector, you must install the Elastic application on your ServiceNow instance and complete additional configuration steps. If you do not do this, an error message displays and you cannot save the new connector. For more information, refer to {kibana-ref}/servicenow-sir-action-type.html[ServiceNow SecOps] and {kibana-ref}/servicenow-action-type.html[ServiceNow ITSM].

Reviewer 2: I think is better to say that you cannot `create` not `save`.

*Automatically close Security cases when pushing new incident to external system*.

[float]
=== Change the default connector

To change the default connector used to send cases to external systems, go to *Cases* -> *Edit external connection* and select the required connector from the Incident management system list.

> TIP: You can also configure which connector is used for each case individually
> (see <<cases-ui-open>>).

Reviewer: ```suggestion
TIP: You can also configure which connector is used for each case individually.
See <<cases-ui-open>>.
```

[role="screenshot"]
image::images/cases-change-default-connector.png[]

[float]
=== Modify connector settings

To change the settings of an existing connector:

. Go to *Cases* -> *Edit external connection*.
. Select the required connector from the *Incident management system list*.
. Click *Update <connector name>*.

> . From the Edit connector flyout, modify the connector fields as required and save the changes.

Reviewer: Actually would like to poll the writing team for the proper preposition for "flyout" - I could be wrong here. 

```suggestion
. In the Edit connector flyout, modify the connector fields as required, then click *Save & close* to save your changes.
```
For the image beneath, crop out the background shadow part - just need to show the flyout.
